app/t/[slug]/request-access/request-access-form.tsx

```
@@ -307,13 +307,31 @@ export function RequestAccessForm({ tenant }: RequestAccessFormProps) {
                             Lot Number <span className="text-mist-gray text-xs">(optional)</span>
                         </Label>
                         <Combobox
-                            options={[
-                                { value: "__none__", label: "— No lot selected —" },
-                                ...lots.map((lot) => ({
-                                    value: lot.id,
-                                    label: `${lot.is_occupied ? "🟡" : "🟢"} Lot ${lot.lot_number}`,
-                                })),
-                            ]}
+                            options={useMemo(() => [
```

Biome (2.4.6)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)